make install-recommend-packages が失敗する

```zsh
❯ cat /etc/os-release
NAME="Amazon Linux"
VERSION="2"
ID="amzn"
ID_LIKE="centos rhel fedora"
VERSION_ID="2"
PRETTY_NAME="Amazon Linux 2"
ANSI_COLOR="0;33"
CPE_NAME="cpe:2.3:o:amazon:amazon_linux:2"
HOME_URL="https://amazonlinux.com/"
```

ややレアな環境だとは思うのですが、以下のコンソール出力で `make  install-recommend-packages` が失敗します


```zsh
❯ sudo make install-recommend-packages
docker-compose exec app composer require doctrine/dbal
Using version ^3.0 for doctrine/dbal
./composer.json has been updated
Running composer update doctrine/dbal
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - barryvdh/laravel-ide-helper is locked to version v2.8.2 and an update of this package was not requested.
    - barryvdh/laravel-ide-helper v2.8.2 requires doctrine/dbal ~2.3 -> found doctrine/dbal[2.3.0-BETA1, ..., 2.12.x-dev] but it conflicts with your root composer.json require (^3.0).

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
make: *** [install-recommend-packages] エラー 2
```

--- a/Makefile
+++ b/Makefile
@@ -12,7 +12,7 @@ create-project:
 	docker-compose exec app php artisan storage:link
 	@make fresh
 install-recommend-packages:
-	docker-compose exec app composer require doctrine/dbal
+	docker-compose exec app composer require doctrine/dbal "^2"
 	docker-compose exec app composer require --dev barryvdh/laravel-ide-helper
 	docker-compose exec app composer require --dev beyondcode/laravel-dump-server
 	docker-compose exec app composer require --dev barryvdh/laravel-debugbar

